modal/sandbox.py

```diff
@@ -165,6 +174,9 @@ async def create(
             Union[str, os.PathLike], Union[_Volume, _CloudBucketMount]
         ] = {},  # Mount points for Modal Volumes and CloudBucketMounts
         pty_info: Optional[api_pb2.PTYInfo] = None,
+        # List of ports to open on the sandbox.
+        # Each tuple is (port, unencrypted).
```

Comment:
Is it confusing to have _unencrypted_? I worry that the negation makes it harder to reason about (i.e. `(8000, False)`. Unless I'm missing something, parameterizing it as `encrypted` just feels simpler.

Reply:
Also, in the interest of minimizing unnecessary verbosity, would `ports` be unclear?

Reply:
Also we chatted about this a little bit but just having `ports: list[int]` would be a lot simpler if there's no compelling usecase for specifying some ports as encrypted and others as unencrypted.

Reply:
We need unencrypted ports to be an option so clients can control TLS behavior if they want. I've refactored to separate variables as discussed offline.